docs/install.md

```diff
@@ -46,7 +46,7 @@
    watch kubectl get pods -o wide
    ```
 
-1. Once the deployment completes, verify Sourcegraph is running by temporarily making the frontend port accessible:
+1. <a name="port-forward"></a>Once the deployment completes, verify Sourcegraph is running by temporarily making the frontend port accessible:
```

Comment:
what does this do?

Reply:
This was left over, I'll delete it. I was just manually creating an anchor so that I could link to it from another document.